docs/getting_started.md

```diff
@@ -47,22 +47,32 @@ If you are not interested in backing up any data, the `--gcs.bucket` flag can si
 * _[Example Kubernetes manifest](../kube/manifests/prometheus.yaml)_
 * _[Example Kubernetes manifest with GCS upload](../kube/manifests/prometheus-gcs.yaml)_
 
-### Query Access
```

Comment:
I would stick to query access section first as it makes sense with the given flow
- backups
- query

Reply:
Will move it after Query Access but before Query layer ... as i think its good to show how the query finds other components in the cluster. Thoughts?

Reply:
perfect